src/vm_core.rs

```diff
@@ -22,14 +23,15 @@ use std::any::Any;
 use std::collections::HashMap;
 use std::{cell::RefCell, rc::Rc};
 
-const GLOBAL_NAMES: [&str; 16] = [
+const GLOBAL_NAMES: [&str; 17] = [
```

Comment:
Do we need to specify the size explicitly here?

Reply:
The size is needed in order to fulfill the `Sized` trait requirement